src/runtime/runtime-debug.cc

```diff
@@ -1033,9 +1032,7 @@ static char* GetProgressMismatchMessage(size_t replayedIndex, uint64_t recordedE
   std::ostringstream os;
   os << "{ \"recorded\": \"" << recorded_text 
      << "\", \"replayed\": \"" << replayed_text
-     << "\", \"pc\": " << (*gProgressCounter - replayedIndex)
-     << ", \"stack\": \"";
-  isolate->PrintCurrentStackTrace(os);
```

Comment:
* I removed it because it crashed in some circumstances. I think it was in a situation where we hit this after an exception was thrown.
* Also, it should be trivial to use the linkerdebug server (or detached replayer for local recordings) to get these data points. I will add relevant samples for it to the `go/linkerdebug` page, when I use the linkerdebug server to run relevant manifests. In another place, we want to might want to look at some on-stack variables to get a sense of what data it is that diverges.